### PR TITLE
feat: add Debugger account id for us-gov-west-1

### PR DESCRIPTION
**Pull request by jigsaw004**
This adds new account id for Debugger Rules repository in AWS Regions us-gov-west-1.

*Issue #, if available:*

*Description of changes:*

*Testing done:*
*No testing required*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [ ] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [ ] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/sagemaker/image_uri_config/debugger.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
                 "us-east-1": "503895931360",
                 "us-east-2": "915447279597",
                 "us-west-1": "685455198987",
-                "us-west-2": "895741380848"
+                "us-west-2": "895741380848",
+                "us-gov-west-1": "515509971035"
```
Comment (jswudi) [inline src/sagemaker/image_uri_config/debugger.json:28]: Do we also need to remove us-gov-west-1 from DEBUGGER_UNSUPPORTED_REGIONS? https://github.com/aws/sagemaker-python-sdk-staging/blob/master/src/sagemaker/fw_utils.py#L51

**Review by ajaykarpur — CHANGES_REQUESTED** [review]
Please also update the corresponding unit test: https://github.com/aws/sagemaker-python-sdk/blob/5f52f95483a310058b193d408034886c353c2685/tests/unit/sagemaker/image_uris/test_debugger.py#L18-L41

And this will also need to be updated: https://github.com/aws/sagemaker-python-sdk/blob/5f52f95483a310058b193d408034886c353c2685/src/sagemaker/fw_utils.py#L51

File: src/sagemaker/image_uri_config/debugger.json (hunk 1 above)
Comment (ajaykarpur) [reply src/sagemaker/image_uri_config/debugger.json:28]: https://github.com/aws/sagemaker-python-sdk/blob/5f52f95483a310058b193d408034886c353c2685/src/sagemaker/fw_utils.py#L51

**Comment by icywang86rui** [comment]
Please fix the commit message based on the contribution guide.

**Comment by icywang86rui** [comment]
Closing due to no activities. Feel free to reopen when you have a chance to address the comments. Thanks!

**Pull request closed without merging**